Update installation doc page (#1389)
OSOceanAcoustics/echopype: docs/source/installation.md

# Installation and examples

## Installation

Echopype is available and tested for Python 3.9-3.11. The latest release can be installed through conda (or mamba, see below) via the [conda-forge channel](https://anaconda.org/conda-forge/echopype):
```shell
# Install via conda-forge
$ conda install -c conda-forge echopype
```

It is available via [PyPI](https://pypi.org/project/echopype):
```shell
# Install via pip
$ pip install echopype
```

:::{note}

> We are working on getting upgraded to 3.12 soon!

Reviewer: ```suggestion
We are working on adding support for Python 3.12 soon!
```
